assets/scss/hero.scss

```diff
@@ -99,9 +100,30 @@
     margin-left: auto;
     margin-right: auto;
   }
-  .hero-wrapper {
+  @media (min-width: $bp-extralarge) {
+    blockquote + p cite {
+      width: 38.3rem;
+    }
+  }
+}
+
+.default-attribution {
```

Comment:
As this is not used on the hero image, move it into its own SCSS file, attributions.scss, and rename the class to "figure-attribution".

Reply:
sure